scripts-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mangoweb/scripts-base",
-  "version": "0.0.10",
+  "version": "0.0.11",
```

Comment:
Doporučuji verze zvyšovat přes `npm version [major,minor,patch]` pro konzistentní `package-lock.json`

Reply:
U ostatních balíčků jsem to udělal. Nevím, proč tady ne… :thinking: Fixed.